doc/configuration.md

```diff
@@ -4,6 +4,15 @@ The Ignition configuration is a JSON document conforming to the following specif
 
 * **ignition** (object): metadata about the configuration itself.
   * **version** (string): the semantic version number of the spec. Must be at least `2.0.0`.
+  * **_config_** (objects): options related to the configuration.
+    * **_append_** (list of objects): a list of the configs to be appended to the current config.
```

Comment:
I think there is a good possibility of having a "merge" in the future. Since that is tougher to define, I've held off on implementing that for now. I prefer the union of actions with options for each since it is very easy to understand how they are applied. If there was a list of configs with merge options for each, it becomes more difficult to define.